src/Opserver.Web/Controllers/MetaController.cs

```diff
@@ -0,0 +1,19 @@
+using Opserver.Views.Shared;
+using Microsoft.AspNetCore.Mvc;
+using System.Net;
+using Microsoft.Extensions.Options;
+using Microsoft.AspNetCore.Authorization;
+using Opserver.Helpers;
+using Opserver.Models;
+
+namespace Opserver.Controllers
```

Comment:
Do we really want to start modifying Opserver? This will make it much harder to pull in upstream changes, which is what caused issues last time.

We could use any page for the healthcheck - /login or something?

Reply:
Paul also suggested to use "/login". I will change our opserver terraform created resource instead.